Libraries/CMSIS/Device/Maxim/MAX32572/Include/ptg_regs.h

```diff
@@ -179,67 +179,67 @@ typedef struct {
 
 /**
  * @ingroup  ptg_registers
- * @defgroup PTG_INTFL PTG_INTFL
- * @brief    Pulse Train Interrupt Flags
+ * @defgroup PTG_STOP_INTFL PTG_STOP_INTFL
+ * @brief    Pulse Train Stop Interrupt Flags
  * @{
  */
-#define MXC_F_PTG_INTFL_PT0_POS                        0 /**< INTFL_PT0 Position */
-#define MXC_F_PTG_INTFL_PT0                            ((uint32_t)(0x1UL << MXC_F_PTG_INTFL_PT0_POS)) /**< INTFL_PT0 Mask */
+#define MXC_F_PTG_STOP_INTFL_PT0_POS                   0 /**< STOP_INTFL_PT0 Position */
```

Comment:
These offsets are more of a pain in the ass to redirect.  I would argue we should do it for backward compatibility, but only if the register file generator can do it.  Is it feasible to add a feature like that to the generator?